lib/explorer/polars_backend/data_frame.ex

```diff
@@ -160,6 +160,15 @@ defmodule Explorer.PolarsBackend.DataFrame do
 
   # Conversion
 
+  @impl true
+  def lazy, do: Explorer.PolarsBackend.LazyDataFrame
+
+  @impl true
+  def to_lazy(df), do: df |> Shared.apply_native(:df_to_lazy) |> Shared.to_dataframe()
```

Comment:
This is losing the groups information, as we talked. :) I created #207 as a follow up.